lib/algos/list/delimiterVersions.js

```diff
@@ -66,6 +69,59 @@ class DelimiterVersions extends Delimiter {
         return params;
     }
 
+    genMDParamsV1() {
+        // return an array of two listing params sets to ask for
+        // synchronized listing of M and V ranges
+        const params = [{}, {}];
+        if (this.parameters.prefix) {
+            params[0].gte = `${DbPrefixes.Master}${this.parameters.prefix}`;
```

Comment:
If you are just concatenating two strings would a+b be faster? Or maybe more readable? Feels like the `${}` might add a little unnecessary noise here.
```
`${DbPrefixes.Master}${this.parameters.prefix}`; // or
DbPrefixes.Master + this.parameters.prefix;
```

Definitely doesn't matter just thought I would bring up the comparison.

Reply:
Yes that could work, I tend to use this pattern when it comes to building strings, also because our beloved linter does not like when we concatenate both string templates and regular strings together, it asks for only string templates or only regular strings (but here it would be happy I think with what you suggest).

Reply:
Right if you are combining a string and variable or combination it makes sense. But 2+ variables together I think should be a + or join. String templates are great for combinations of strings and variables though.

Reply:
updated